src/cephadm/cephadm.py

```diff
@@ -8145,7 +8145,7 @@ def _extract(sock: socket.socket, offset: int) -> str:
 
     dec_mask = sum([bin(int(i)).count('1')
                     for i in dq_mask.split('.')])
-    return '{}/{}'.format(addr, dec_mask)
+    return '{}'.format(addr)
```

Comment:
for flake8
```
cephadm.py:8146:5: F841 local variable 'dec_mask' is assigned to but never used
1     F841 local variable 'dec_mask' is assigned to but never used
```

Reply:
dec_mask is nowhere used apart from the following line -https://github.com/ceph/ceph/blob/main/src/cephadm/cephadm.py. We can remove this variable as there is no dependency on it for functionality of any ceph features.

Reply:
yes, I'd just remove it as part of this change